src/runtime/runtime.cs

```diff
@@ -80,16 +80,22 @@ public static IntPtr GetProcAddress(IntPtr dllHandle, string name)
     /// </summary>
     public class Runtime
     {
+        // C# compiler copies constants to the assemblies that references this library.
+        // We needs to replace all public constants to static readonly fields to allow 
+        // binary substitution of different Python.Runtime.dll builds in a target application.
+
+        public static readonly int UCS = _UCS;
```

Comment:
Please change these to properties. Allows us to make this fully dynamic in the future without breaking user code.